eclipse-scout-core/src/testing/JasmineScoutUtil.ts

```diff
@@ -101,11 +101,32 @@ export const JasmineScoutUtil = {
   },
 
   /**
-   * If a ajax call is not mocked, this fallback will be triggered to show information about which url is not mocked.
+   * If an ajax call is not mocked, this fallback will be triggered to show information about which url is not mocked.
    */
   captureNotMockedCalls() {
     jasmine.Ajax.stubRequest(/.*/).andCallFunction((request: JasmineAjaxRequest) => {
       fail('Ajax call not mocked for url: ' + request.url + ', method: ' + request.method);
     });
+  },
+
+  /**
+   * Calls the given mock as soon as a hybrid action with the given actionType is called.
+   * The mock is called asynchronously using setTimeout to let the runtime code add any required event listeners first.
+   *
+   * The mock may return an object with [id, widget] if the action is supposed to create widgets.
+   * The id needs to contain the action id and the id for the widget used in the hybrid action (`${actionId}${widgetId}`).
```

Comment:
This id-format is not always correct, is it? Or it needs to be pointed out that the widget id may be empty. If one wants to mock e.g. an openForm action, the id needs to be `event.data.id`.

The id needs to be the same as created by the `IHybridAction` (see `AbstractHybridAction.addWidget(IWidget)`, `AbstractHybridAction.addWidgets(Map<String, IWidget>)`)